submissions/amashoshyna/js-dom/Liddell777/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,117 @@
+                                    <img src="./img/menupopup/app.png" alt="apps icon">
+                                    Apps
+                                </a>
+                                <a class="mainpopup__link" href="#">
+                                    <img src="./img/menupopup/docs.png" alt="docs icon">
+                                    Document
+                                </a>
+                                <input type="checkbox" id="mainpopup__input" class="mainpopup__item-input" />
+                                <label for="mainpopup__input" class="mainpopup__item-label" tabindex="0"></label>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/cloud-network.png" alt="cloud network">
+                                    Cloud
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/gear.png"  alt="gear">
+                                    Gear
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/group.png"  alt="group">
+                                    Group
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/music-folder.png" alt="music folder">
+                                    Music
+                                </a>
+                                <a class="mainpopup__link mainpopup__link--hidden" href="#">
+                                    <img src="./img/menupopup/menumore/shield.png"  alt="shield">
+                                    Shield
+                                </a>
+                            </div>
+                        </div>
+                    </li>
+                    <li class="menu__item">
+                        <a class="menu__link-bell" href="#"><img src="img/menu/bell.png" alt="bell"></a>
+                    </li>
+                    <li class="menu__item">
+                        <a class="menu__link-user" href="https://github.com/Liddell777"><img src="img/menu/45641044.jpg" alt="picture"></a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <div class="container">
+                <div class="search__panel">
+                    <input class="container__input" type="search" placeholder="Feed me a leather sack">
+                    <input type="submit" class="container__btn" value="">
+                </div>
+            </div>
+        </main>
+        
+</body>
+</html>
```

Comment:
Could you please add the empty line after the last tag

Reply:
Hi, I'm sorry, I didn't really understand what empty line do you mean? what line of code with error? thanks

Reply:
GitHub puts a mark in such a case. It is important that every file has an empty line at its end.
Your IDE most likely has a setting to take care about it for you.
![image](https://user-images.githubusercontent.com/8686631/99372247-926b2e00-28c8-11eb-93ff-bcad8f117899.png)